### DISCUSSION TITLE
Any ways to specify server-side-only translations that are not included in the frontend bundle?

### DISCUSSION THREAD

**Discussion by raf202** [nuxt-modules/i18n#3000] [Q&A]
Using Nuxt 3, we're specifying some translation files that are only used in our backend API (server folder) and returned via API.
The problem is that they're also getting included and loaded in the frontend code, even though nothing on our frontend calls these directly. These files can have a LOT of translation strings and become huge with size, so we're trying to make it load only on the server side to prevent our frontend application becoming too resource consuming.

Any ideas how to achieve this?

**Comment by kazupon** [comment]
Thank you for your reporting.
This is not github issue, is question or consultation.
I'll move to github dicussions

**Comment by Sun-ZhenXing** [comment]
Same problem, I want the backend translations to use separate translation contexts.

Currently I'm implementing backend translations in my project via self-implemented translations, using a custom `mapping.json` to define these mappings, which is separate from the frontend translation files.

I hope this ugly approach will be optimized for the next release.

**Comment by arily** [comment]
I have this need too, or solve the root problem that we may need multiple locale roots, i.e for admin panels (separate root), error codes (server only), etc.
